index.bs

```diff
@@ -63,6 +63,10 @@ The following keys must be considered valid:
 * <a href="#key-devtools_page">`devtools_page`</a>: optional
 * <a href="#key-externally_connectable">`externally_connectable`</a>: optional
 
+The following keys must be considered valid in manifest v3:
```

Comment:
```suggestion
The following keys must be considered valid in Manifest V3:

```

* Insert a newline between the list and paragraph for consistency.
* "Manifest V3" is a proper noun, so it should be uppercased. The canonical casing Chrome has been using is to capitalize both the M and V. I'd prefer to default to that pattern unless and until we adopt a different pattern in this group.